client/state/sites/plans/actions.js

```diff
@@ -127,7 +127,7 @@ export function fetchSitePlansCompleted( siteId, data ) {
 	let plansData = omit( data, '_headers' );
 	const { product_id } = personalPlan;
 
-	if ( ! has( plansData, product_id ) ) {
+	if ( ! isEmpty( plansData ) && ! plansData.hasOwnProperty( product_id ) ) {
```

Comment:
That's right, we could write the expression as
```js
! ( isEmpty( plansData ) || plansData.hasOwnProperty( product_id ) )
```
or
```js
! ( isEmpty( plansData ) || has( plansData, product_id ) )
```

I personally like double negatives. They have a certain _je ne sais quoi_... but I'll change it and add back `has`.

![6f9f07f1cc4f49da715121bf413ff50d91dea6e46c408fdbb8b0f56bd6730d0e](https://cloud.githubusercontent.com/assets/233601/16168994/223fd2ca-34f3-11e6-8ba1-a27b6ffbb74b.jpg)